pkg/services/ngalert/api/tooling/api.json

```diff
@@ -4644,7 +4664,76 @@
   "gettableAlerts": {
    "description": "GettableAlerts gettable alerts",
    "items": {
-    "$ref": "#/definitions/gettableAlert"
+    "$ref": "#/definitions/gettableAlert",
+    "type": "object"
+   },
+   "type": "array"
+  },
+  "gettableGrafanaSilence": {
```

Comment:
Neat, but counterintuitive - deleting the `// swagger:model` comment makes the type start appearing in the doc :upside_down_face: